web/src/stories/docs/plugins/AppxListKnownPlugins.story.mdx

```diff
@@ -0,0 +1,25 @@
+import { Meta } from '@storybook/addon-docs/blocks';
+
+<Meta
+  title="Docs/Plugins/A1. List of Known Plugins"
+  parameters={{
+    viewMode: 'docs',
+    previewTabs: { canvas: { hidden: true } },
+    canvas: { disable: true },
+  }}
+/>
+
+# Plugin List
+
+Below you will find a list of unoffical Octant plugins. This list is provided purely to help folks find available Octant plugins
+easier. These plugins may be in various states of development. If you are having an issue running a plugin please contact
+the plugin authors directly using their prefered method for reporting issues.
+
+ * Plugin Name - Link
+
+## How to submit
+
+If you would like your plugin to be listed here, please create a Pull Request and update the file referenced below to include
+the name and link to your plugin. Plugins should be ordered alphabetically.
+
+    File: web/src/stories/docs/plugins/AppxListKnownPlugins.story.mdx
```

Comment:
It would be good if list of plugins is inside separate `json` file, so it can later be retrieved directly  from the UI

Reply:
Out of scope for this change.